json-el/src/main/scala/io/zeebe/msgpack/el/JsonObject.scala

```diff
@@ -65,9 +66,9 @@ case class JsonString(value: DirectBuffer) extends JsonObject with JsonConstant
   token.setValue(value, 0, value.capacity())
 }
 
-case class JsonPath(value: String) extends JsonObject {
-  val query: JsonPathQuery = new JsonPathQueryCompiler().compile(value)
-  val variableName = query.getTopLevelVariable
+case class JsonPath(variableName: DirectBuffer, path: List[String]) extends JsonObject {
+  val jsonPath = "$." + (bufferAsString(variableName) :: path).mkString(".")
+  val query: JsonPathQuery = new JsonPathQueryCompiler().compile(jsonPath)
```

Comment:
We still need the top level variable name from the json query at the in/out mappings right? Or can we remove it

Reply:
The `topLevelVariable` is still used for input/output mappings.